block_server.py

```diff
@@ -664,15 +657,17 @@ def consume_write_queue(self):
             self.load_config, ("configname",), "LOADING_CONFIG")
         """
         while True:
-            while len(self.write_queue) > 0:
-                with self.write_lock:
+            with self.write_lock:
+                if self._filewatcher is not None: self._filewatcher.pause()
```

Comment:
Pop the item off the queue with the write lock then release the lock and process the command.